README.md

```diff
@@ -7,6 +7,22 @@ A boilerplate-free Scala library for loading configuration files
 
 ![](http://i.imgur.com/S5QUS8c.gif)
 
+## Table of Contents
+- [Why](#why)
+- [Not Yet Another Configuration Library](#not-yet-another-configuration-library)
+- [Add PureConfig to your project](#add-pureconfig-to-your-project)
+- [Use PureConfig](#use-pureconfig)
+- [Supported types](#supported-types)
+- [Extends the library to support new types](#extend-the-library-to-support-new-types)
+- [Override behaviour for types](#override-behaviour-for-types)
+- [Example](#example)
+- [Whence the config files](#whence-the-config-files)
+- [Contribute](#contribute)
+- [License](#license)
+- [Special thanks](#special-thanks)
+
+
+<a name="why" />
```

Comment:
You don't need to add these anchors, as GitHub adds them automatically (see https://github.com/melrief/pureconfig/blob/master/README.md#why, for example).